snapcraft/plugins/v1/go.py

```diff
@@ -14,7 +14,17 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
-"""The go plugin can be used for go projects using `go get`.
+"""The go plugin can be used for go projects.
+
+The plugin supports Go Modules if a go.mod definition is found in the
```

Comment:
perhaps something like s/in the sources out of the box/in the root of the part's source/ ?